config/config.exs

```diff
@@ -17,6 +17,24 @@ config :mantra, Mantra.Endpoint,
   pubsub: [name: Mantra.PubSub,
            adapter: Phoenix.PubSub.PG2]
 
+config :ueberauth, Ueberauth,
+  providers: [
+    github: {Ueberauth.Strategy.Github, []}
+  ]
+
+config :ueberauth, Ueberauth.Strategy.Github.OAuth,
+  client_id: System.get_env("GITHUB_CLIENT_ID"),
```

Comment:
how do you manage env variables locally? i got spoiled by figaro on ruby hahaha.